src/briefcase/commands/build.py

```diff
@@ -14,13 +17,46 @@ def add_options(self, parser):
         self._add_update_options(parser, context_label=" before building")
         self._add_test_options(parser, context_label="Build")
 
-    def build_app(self, app: BaseConfig, **options):
+    def build_app(self, app: AppConfig, **options):
         """Build an application.
 
         :param app: The application to build
         """
         # Default implementation; nothing to build.
 
+    def verify_template_compatability(self, app: AppConfig):
+        """Verify the template satisfies the Command's requirements."""
+        self.logger.info("Verifying template compatability...", prefix=app.app_name)
+
+        with self.input.wait_bar("Verifying Briefcase version support..."):
```

Comment:
Not sure this needs a wait bar - it should be near instantaneous, and in the "normal" case there's nothing actionable from the perspective of the user.

Reply:
It is true it's near instantaneous but I'm considering it more to be communicating the action that is taking place and the `done` shows it succeeded. This isn't too dissimilar from a lot of other uses of the Wait bar (for instance, verifying glibc)...although, in this scenario, I think this would be clearer is additional aspects of the template were being verified than just if Briefcase is compatible.